Packs/Cloudflare_New/Integrations/CloudflareNew/CloudflareNew.yml

```diff
@@ -0,0 +1,59 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Cloudflare_new
+  version: -1
+configuration:
+- additionalinfo: API Key generated by cloudflare. Read access to all resources required.
+  display: Bearer token
+  name: API-key
+  required: true
+  type: 4
```

Comment:
please use type 9 instead of 4.
it should look something like this:
- displaypassword: API Key
  additionalinfo: API Key generated by cloudflare. Read access to all resources required.
  name: credentials
  required: true
  hiddenusername: true
  type: 9
  
  
  and when retrieving it in the code it should be something like:
  apikey = demisto.params().get("credentials", {}).get("password")